catalog/solr/pom.xml

```diff
@@ -30,6 +30,11 @@
                 <artifactId>joda-time</artifactId>
                 <version>2.7</version>
             </dependency>
+            <dependency>
+                <groupId>commons-fileupload</groupId>
+                <artifactId>commons-fileupload</artifactId>
+                <version>1.3.2</version>
```

Comment:
The fact that there is a transitive dependency has no bearing on using the property here. We use the property everywhere else, including in the `ddf.platform.solr:solr` module where we specifically note that it is because of Solr's transitive dependencies: https://github.com/codice/ddf/blob/master/platform/solr/pom.xml#L46

Please use the version property here.